packages/core/useStepper/index.ts

```diff
@@ -0,0 +1,83 @@
+import { computed, ref } from 'vue-demi'
+
+export function useStepper<T extends string>(steps: readonly T[], initial?: T) {
+  const index = ref(steps.indexOf(initial ?? steps[0]))
+  const current = computed(() => steps[index.value])
+  const isFirst = computed(() => index.value === 0)
+  const isLast = computed(() => index.value === steps.length - 1)
+
+  function backTo(step: T) {
+    if (currentStepIsAfter(step))
+      goTo(step)
+  }
```

Comment:
It would feel a bit weird to me that:

```ts
const s = useStepper([1,2,3,4])
s.goTo(2)
// ...
s.backTo(3)
// and s stays on 2
```

In your case, the `disabled` already serve as a guard for it.

Reply:
Yeah, what I mean is the guard should be done on userland when calling `goTo`. Since there is only one line of addition in `backTo`, the trade-off of saving one line but introducing an additional abstraction layer that users/readers need to understand is a bit less worthy to me. I'd suggest we remove it, and let users define it when needed.

Reply:
I understand, I still think it's worth it since I used it in every wizard I made, but I'll remove it since it's not that bad to create another function for that